### DISCUSSION TITLE
[Feature] Add immich to portainer app templates

### DISCUSSION THREAD

**Discussion by Lauritz-Tieste** [immich-app/immich#11482] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I think we should add immich to the app templates at Portainer to make it accessible to more people.

I haven't researched this extensively yet, but I think you just need to add it to this repository: https://github.com/portainer/templates

**Comment by bo0tzz** [comment]
We don't tend to support things like that ourselves, but feel free to create an unofficial/community template.
